### ISSUE TITLE
[IDEA] : group operation : delete / download all wikis 

### ISSUE THREAD

**Issue by Telumire** [simonbaird/tiddlyhost#289]
It would be great to have a way to download all of my wikis at once every once in a while, for backups purpose. Deleting all wikis could be useful too to clean up one's account / free space.

**Comment by simonbaird** [comment]
Yeah, it's a good idea. There's #265 open already, but for this one I'm imagining a UI with a checkbox on each row, so you can perform actions on the selected sites, e.g. "Delete selected".
